aiopyql/table.py

```diff
@@ -428,8 +428,13 @@ async def insert(self, **kw):
                 cols = f'{cols}, '
                 vals = f'{vals}, '
             cols = f'{cols}{col_name}'
+            #postgres array
+            if str(self.database) == 'postgres' and col.type == str and (kw[col_name][0], kw[col_name][-1]) == ('[',']'):
+                print('hi')
```

Comment:
This is a friendly, but probably un-required print statement

Reply:
@AdhTri-001

Reply:
Lmao that was for testing